terraform-infra-approvals/pre-shared-infrastructure.json

```diff
@@ -18,7 +18,11 @@
     {"type": "azurerm_role_assignment"},
     {"type": "azurerm_disk_encryption_set"},
     {"type": "azurerm_key_vault_key"},
-    {"type": "azurerm_monitor_diagnostic_setting"}
+    {"type": "azurerm_monitor_diagnostic_setting"},
+    {"type": "azurerm_virtual_machine_extension"},
+    {"type": "azurerm_dev_test_global_vm_shutdown_schedule"},
+    {"type": "azurerm_log_analytics_solution"},
+    {"type": "azurerm_log_analytics_workspace"}
```

Comment:
you may not create your own workspace, you must use the central ones which have discounts applied